app/presenters/finder_presenter.rb

```diff
@@ -55,6 +55,10 @@ def filters
     facets.filters
   end
 
+  def feed_disabled?
```

Comment:
I had that originally but after chatting with @elliotcm his advice was to use the method to modify what is the default for Finders.